.github/workflows/format.yml

```
@@ -0,0 +1,37 @@
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+
+      - name: Format code
+        run: |
+          cd expo-app
+          npx prettier . --write
+
+      - name: Commit and push changes
+        if: github.event.pull_request.head.repo.full_name == github.repository
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)